signal/src/lib.rs

```diff
@@ -286,19 +287,21 @@ pub mod pallet {
 				ensure!(proposal_deposit >= T::MinProposalDeposit::get(), Error::<T>::DepositInsufficient);
 			}
 			// Check if all parameters are combinable:
-			if unit == Unit::Person && scale == Scale::Quadratic {
-				return Err(Error::<T>::WrongParameter)?;
-			};
-			if unit == Unit::Token && majority == Majority::Absolute {
-				return Err(Error::<T>::WrongParameter)?;
-			};
-			// TODO: quorum validation. Should work only for the Unit:Person
+			match unit {
+				Unit::Person => {
```

Comment:
what if the voting account is not a person but a machine?

Reply:
Let's think of the other name, I agree it's not the best one.